selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestWorkspaceServiceClient.java

```diff
@@ -64,53 +66,43 @@ private String getBaseUrl() {
   }
 
   /** Returns the list of workspaces names that belongs to the user. */
-  public List<String> getAll() throws Exception {
-    List<WorkspaceDto> workspaces =
-        requestFactory.fromUrl(getBaseUrl()).request().asList(WorkspaceDto.class);
+  public List<String> getAll(TestUser owner) throws Exception {
```

Comment:
Why don't inject `TestHttpJsonRequestFactory` for non-default user?

Reply:
Well, we can do it, but the code will be less clear so as it requires creation of additional instances of TestWorkspaceServiceClient class with specific TestHttpJsonRequestFactory.